system/default_ad_api/schema/default_ad_api.schema.json

```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for default_ad_api",
+  "type": "object",
+  "definitions": {
+    "default_ad_api_node": {
+      "type": "object",
+      "properties": {
+        "update_rate": {
+          "type": "number",
+          "default": 10.0,
+          "description": "Update rate for the node."
+        },
+        "require_accept_start": {
+          "type": "boolean",
+          "default": false,
+          "description": "Require acceptance to start."
+        },
+        "stop_check_duration": {
+          "type": "number",
+          "default": 1.0,
+          "description": "Duration for stop check."
+        },
+        "stop_distance": {
+          "type": "number",
+          "default": 1.0,
+          "description": "Stop distance."
+        },
+        "stop_duration": {
+          "type": "number",
+          "default": 1.0,
+          "description": "Stop duration."
+        }
+      }
+    }
+  },
+  "properties": {
+    "/default_ad_api/node/autoware_state": {
```

Comment:
Is it possible to make this node name part (as well as the other nodes) match all patterns? This is because namespaces and node names may differ depending on the tool.

Reply:
I updated the schema file.

Reply:
Hi @isamu-takagi,

Thank you for your review. The schema file has been updated. I would appreciate it if you could review the changes in case I misunderstood you.